.github/workflows/wot-td.yml

```diff
@@ -133,7 +133,7 @@ jobs:
 
     strategy:
       matrix:
-        platform: [ubuntu-lates]
+        platform: [ubuntu-latest]
```

Comment:
@lu-zero I noticed that there is a small spelling mistake here, which prevents the CI from completing. Is it fine to apply the fix in this PR? Or should I open a separate one?